src/main.cpp

```diff
@@ -7,6 +7,7 @@
  * openblack is licensed under the GNU General Public License version 3.
  *******************************************************************************/
 
+// TODO: Cause a clang-tidy error
```

Comment:
warning: missing username/bug in TODO [google-readability-todo]

```suggestion
// TODO(bwrsandman): Cause a clang-tidy error
```